CMakeLists.txt

```diff
@@ -65,6 +65,30 @@ else ()
         CACHE STRING "Installation path for plugins" FORCE)
 endif ()
 
+if ( NOT ZEEK_LOCAL_STATE_DIR )
+    set(ZEEK_LOCAL_STATE_DIR ${ZEEK_ROOT_DIR})
+endif ()
+
+if ( NOT ZEEK_ETC_INSTALL_DIR )
+    set(ZEEK_ETC_INSTALL_DIR ${ZEEK_ROOT_DIR}/etc)
+endif ()
+
+if ( NOT ZEEK_STATE_DIR )
+    set(ZEEK_STATE_DIR ${ZEEK_LOCAL_STATE_DIR}/var/lib/zeek)
+endif ()
+
+if ( NOT ZEEK_SPOOL_DIR )
+    set(ZEEK_SPOOL_DIR ${ZEEK_LOCAL_STATE_DIR}/spool)
+endif ()
+
+if ( NOT ZEEK_LOG_DIR )
+    set(ZEEK_LOG_DIR ${ZEEK_LOCAL_STATE_DIR}/logs)
+endif ()
+
+install(DIRECTORY DESTINATION ${ZEEK_STATE_DIR})
```

Comment:
Create `etc/` as well here, just to be safe?

Reply:
Ah right, good call. It currently gets created only when zeekctl or zkg are enabled, but not otherwise.